src/main/resources/fileTemplates/testMeIncludes/TestMe macros.java.ft

```diff
@@ -86,7 +88,7 @@ assertEquals(result#{if}($TestSubjectUtils.isJavaFuture($method.returnType)).get
 ##
 #macro(renderMockStubs $method $testedClass)
 #foreach($field in $testedClass.fields)
-#if($MockitoMockBuilder.isMockable($field, $testedClass))
+#if($mockBuilder.isMockable($field, $testedClass))
 #foreach($fieldMethod in $field.type.methods)
 #if($fieldMethod.hasReturn() && $TestSubjectUtils.isMethodCalled($fieldMethod,$method))
         when($field.name.${fieldMethod.name}($MockitoMockBuilder.buildMockArgsMatchers(${fieldMethod.methodParams},"Java"))).thenReturn($TestBuilder.renderReturnParam($method,$fieldMethod.returnType,"${fieldMethod.name}Response",$replacementTypes,$defaultTypeValues));
```

Comment:
`$MockitoMockBuilder` should be replaced with `$mockBuilder`. there are more such occurrences in this file that should be find-n-replaced.
 _buildMockArgsMatchers_ and some additional methods should be added to the new interface - MockBuilder.
Great to see this being refactored :heart_eyes: